charts/dependency-track/values.yaml

```diff
@@ -20,6 +20,8 @@ common:
     automount: false
 
 apiServer:
+# -- The type of deployment. Can be either deployment or statefulSet.
+  deploymentType: statefulSet
```

Comment:
Please add this field to `values.schema.json`: https://github.com/DependencyTrack/helm-charts/blob/2c578a93fea56ea545ba9dae4c669a26b6a50d72/charts/dependency-track/values.schema.json#L59-L131

Consider making this field an `enum` so that users will know sooner if they have a typo: https://json-schema.org/understanding-json-schema/reference/enum

Reply:
fixed in [cd8ec34](https://github.com/DependencyTrack/helm-charts/pull/102/commits/cd8ec34df593ff8571194caec14874be95e55210)